### DISCUSSION TITLE
采用export_model导出文字检测模型，精度下降厉害，请问是否export_model.py需要修改参数

### DISCUSSION THREAD

**Discussion by jingmingtao** [PaddlePaddle/PaddleOCR#12664] [Q&A]
采用export_model导出文字检测模型，精度下降厉害，请问是否export_model.py需要修改参数

**Comment by justajustin** [comment]
Infer的处理和predict的处理不同，导致两个代码结果差别很大。
我下载训练模型，在infer_rec上都是乱码，predict_rec上正常

**Comment by SWHL** [comment]
如果问题仍然存在，可以尝试最新代码
